### DISCUSSION TITLE
MultiselectCombobox component issues

### DISCUSSION THREAD

**Discussion by LechK** [twilio-labs/paste#2885] [Ideas] [answered]
Hello. I have noticed, that `MultiselectCombobox` component is not working properly with form validation. If we give it `required`, it will require inputValue (what we type to search for item) `onSubmit`, not selected Items length. Maybe better would be require `minLength` to validate that?

**Comment by nkrantz** [comment]
Hey @LechK - good find. I created a ticket for us to work on this and we'll update this thread with more info. Please check back in in the new year!

**Comment by SiTaggart** [comment, marked as answer]
The required prop should now work correctly with both Combobox and Multiselect comboobx.
